ProcessMaker/processmaker
# Bugfix/FOUR-4810: When I import a process that has assignment rules by user ID, the variable is lost in the configuration

## Issue & Reproduction Steps

**Issue 1:**
When you import a process that have a task with assignment rules configured as **User By Id** with some value, that ID value is lost after importing.
- Create a process with a task
- Select **By User Id** in Assignment Type select under Assignment Rules and in Variable name add some User ID
- Save and then export the process
- Import the process
- Go to edit the process and see the variable name is missing.

**Issue 2:**
When you import a process that have a task with assignment rules configured as **Users / Groups** with some user, the Assignment type select not selecting **User/Groups** option after importing.
- Create a process with a task
- Select **Users / Groups** in Assignment Type select under Assignment Rules and select some User
- Save and then export the process
- Import the process
- Go to edit the process and see the select does not have **Users / Groups** option selected.

## Solution
**Solution for Issue 1:**
- Added a conditional to not remove assignments configured as **user by id**.

**Solution for Issue 2:**
- Changed the attribute from **user** to **user_group** in controller because the exporting file adds the attribute user_group instead user

## How to Test
- Create a process with a 2 task
- In the first select **By User Id** in Assignment Type select under Assignment Rules and in Variable name add some User ID
- In the second task select **Users / Groups** in Assignment Type select under Assignment Rules and select some User
- Save and then export the process
- Import the process
- Go to edit the process and see the variable name for the task 1 have the value (value are not missed) and for task 2 the select Assignment Type is setted.

Run test under tests/Feature/Processes/ExportImportTest.php 

**Working video**

https://user-images.githubusercontent.com/90727999/145090897-2c9bab30-bce3-4b8a-91d0-f16acaf0b9d8.mov


## Related Tickets & Packages
- [FOUR-4810](https://processmaker.atlassian.net/browse/FOUR-4810)

## Code Review Checklist
